compiler/daemon/daemon-common/src/org/jetbrains/kotlin/daemon/common/CompilationOptions.kt

```diff
@@ -86,16 +86,16 @@ class IncrementalCompilationOptions(
 
     override fun toString(): String {
         return "IncrementalCompilationOptions(" +
-               "super=${super.toString()}, " +
-               "areFileChangesKnown=$areFileChangesKnown, " +
-               "modifiedFiles=$modifiedFiles, " +
-               "deletedFiles=$deletedFiles, " +
-               "classpathChanges=$classpathChanges, " +
-               "workingDir=$workingDir, " +
-               "multiModuleICSettings=$multiModuleICSettings, " +
-               "usePreciseJavaTracking=$usePreciseJavaTracking" +
-               "outputFiles=$outputFiles" +
-               ")"
+                "super=${super.toString()}, " +
+                "areFileChangesKnown=$areFileChangesKnown, " +
+                "modifiedFiles=$modifiedFiles, " +
+                "deletedFiles=$deletedFiles, " +
+                "classpathChanges=${classpathChanges::class.simpleName}, " +
```

Comment:
Nit: Probably would be better to override `.toString()` in `ClasspathChanges` classes and provide additional info when possible.

Reply:
In general, yes, but in this case `classpathChanges.toString()` would be a bit long (it would contain file paths of all classpath entries), whereas here we just need to know the type of `classpathChanges`.

I'll use your suggestion later if there is more we need to know about `classpathChanges`.